### DISCUSSION TITLE
QTable: Loading state with additional columns, indicator does not go all across

### DISCUSSION THREAD

**Discussion by Dirk-** [quasarframework/quasar#9874] [General - Components / Directives / etc] [answered]
I have a table with some data columns and two additional columns with buttons. The loading state indicator does not go all across the table, it only uses as many columns as the `columns` array has.

Can I include the buttons in the `columns` array somehow? Or is there another option to make the loading indicator go all across?

Please see my CodePen for demonstration:

https://codepen.io/Dirk-/pen/NWjPveq

Thanks
Dirk

**Comment by pdanpdan** [comment, marked as answer]
`table-colspan`

**Reply by Dirk-** [reply]
Yes, thanks! In my case:

```
    <q-table
      title="Treats"
      :data="data"
      :columns="columns"
      color="primary"
      row-key="name"
      :loading="loading"
      table-colspan="6"
    >

```
